src/services/navigation/types.ts

```diff
@@ -3,3 +3,4 @@ import pb from '../../gen/service/navigation/v1/navigation_pb';
 export type ModeMap = pb.ModeMap;
 export type Waypoint = pb.Waypoint.AsObject;
 export type NavigationPosition = pb.GetLocationResponse.AsObject;
+export type Path = pb.Path.AsObject;
```

Comment:
also export this type [in src/main.ts](https://github.com/viamrobotics/viam-typescript-sdk/blob/b6c61ff0a030fff17f279ecd875f1ac28109b92a/src/main.ts#L336-L341) as well if you want users to be able to import it. if that's not important than disregard

Reply:
Good catch - added!